docs/mine/lotus/split-markets-miners.md

```diff
@@ -482,3 +482,13 @@ As soon as the message is confirmed, clients will know to look for the node iden
 ```shell
 lotus-miner actor set-addrs <NEW_MULTIADDR>
 ```
+9. update the Addresses section of the markets config.toml,Update this section as same  as  mining/sealing/proving node
+> [Addresses]  
+  \#PreCommitControl = ["f00XX1"]  
+  \# CommitControl = ["f00XX2"]  
+    TerminateControl = []  
+  \#  DealPublishControl = ["f00XX3"] 
+    DisableOwnerFallback = false  
+    DisableWorkerFallback = false 
```

Comment:
````suggestion

```toml
[Addresses]  
  \#PreCommitControl = ["f00XX1"]  
  \# CommitControl = ["f00XX2"]  
    TerminateControl = []  
  \#  DealPublishControl = ["f00XX3"] 
    DisableOwnerFallback = false  
    DisableWorkerFallback = false 
```
````

Code examples should be in code blocks.